scripts/fast-reboot-dump.py

```diff
@@ -11,39 +11,58 @@
 import argparse
 import syslog
 import traceback
+import ipaddress
 
 
 ARP_CHUNK = binascii.unhexlify('08060001080006040001') # defines a part of the packet for ARP Request
 ARP_PAD = binascii.unhexlify('00' * 18)
 
-def generate_arp_entries(filename, all_available_macs):
+def is_ipv4_address(ipaddr):
```

Comment:
Actually there are Python lib that returns the version of the ip address as indicated ealier. It's better to use it. Here is an example:

https://github.com/Azure/sonic-mgmt/blob/master/tests/common/fixtures/advanced_reboot.py#L136